client/webserver/webserver_test.go

```diff
@@ -99,11 +98,33 @@ func (r *TReader) Read(p []byte) (n int, err error) {
 func (r *TReader) Close() error { return nil }
 
 type TConn struct {
-	msg []byte
+	msg       []byte
+	reads     [][]byte      // data for ReadMessage
+	respReady chan []byte   // signal from WriteMessage
+	close     chan struct{} // Close tells ReadMessage to return with error
```

Comment:
This appears to be unused.

Reply:
This field gets used in ReadMessage.

Although this diff is from one of my commits in https://github.com/decred/dcrdex/pull/186 that the present PR is currently rebased on.